rosbridge_library/src/rosbridge_library/capabilities/call_service.py

```diff
@@ -81,6 +81,7 @@ def call_service(self, message):
         fragment_size = message.get("fragment_size", None)
         compression = message.get("compression", "none")
         args = message.get("args", [])
+        timeout = message.get("timeout", 5.0)
```

Comment:
So the previous default was to never time out. Should this continue to be the case?

Reply:
If we want to retain current behavior, it should by default never timeout.

Reply:
Though I think we could change the defaults for rolling and future distros. @sea-bass what do you think about branching out for Jazzy ?

Reply:
I'm OK either way.

It would be the "right" thing to do to branch out for Jazzy, but I think we still have a few more months where it's okay to land breaking changes to Jazzy.

As an unofficial rule, I would say we can keep putting changes into Jazzy until May 2025 where you may see more people upgrading from ROS 1.